src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -397,6 +397,11 @@
   "skyblocker.config.mining.crystalsWaypoints.findInChat.@Tooltip": "When in crystal hollows read the chat to see if coordinates are sent and extract these to show as waypoint or on the map",
   "skyblocker.config.mining.crystalsWaypoints.shareFail": "Can only share waypoints you have found.",
 
+  "skyblocker.config.mining.glacite.coldOverlay": "Glacite Tunnels",
```

Comment:
Remove the `coldOverlay` reference in the translation string key. Also move it underneath the drill fuel so that its in alphabetical order with the other translation strings